Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -9,14 +9,20 @@ include(["utils", "functions", "filesystem", "files"]);
 */
 Wine.prototype.DXVK = function () {
     print("NOTE: you need a driver that supports Vulkan enough to run DXVK");
-    print("NOTE: wine version should be greater or equal to 3.5");
-
-    var dxvkVersion = "0.53";
+    print("NOTE: wine version should be greater or equal to 3.10");
+    
+    var releaseFile = new Resource()
+        .wizzard(this.wizzard())
+        .url("https://github.com/doitsujin/dxvk/blob/master/RELEASE")
+        .name("RELEASE.txt")
+        .get();
+    
+    var dxvkVersion = function cat(this.prefixDirectory() + "RELEASE.txt");
 
     var setupFile = new Resource()
         .wizard(this.wizard())
         .url("https://github.com/doitsujin/dxvk/releases/download/v" + dxvkVersion + "/dxvk-" + dxvkVersion + ".tar.gz")
-        .checksum("df448f1a7b842773f4d826a86eeba44c937187b0")
+        .checksum("f3965e1eddaf96ee7bf7f6c1c3e71e59e8f90d10")
```

Comment:
I think you should maybe delete the checksum, so that if dxvk version changes, you have nothing to do.